Header header

# Regular 16 sbus channels with 11 bit value range each
uint16[16] channels

# Digital channels
bool digital_channel_1
bool digital_channel_2

# Frame lost flag
bool frame_lost

# Failsafe flag
bool failsafe
